### PR TITLE
[daint-gpu] Create LAMMPS-22Aug2018-CrayGNU-18.08-cuda-9.1-pat-7.0.2.eb

### PR DESCRIPTION
**Pull request by twrobinson**

### CODE REVIEW

**Comment by lucamar** [comment]
retest this please

**Comment by gppezzi** [comment]
ok to test

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-22Aug2018-CrayGNU-18.08-cuda-9.1-pat-7.0.2.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+prebuildopts += ' make -f Makefile.gpu && popd && '
+#create Makefile.omp and correct Makefile.mpi
+prebuildopts += ' sed -e \'s/mpicxx/CC -fopenmp/\' ./MAKE/Makefile.mpi > ./MAKE/Makefile.omp && '
+prebuildopts += ' sed -i -e \'s/mpicxx/CC/\' ./MAKE/Makefile.mpi && '
+buildopts = [ ' mpi ', ' omp ' ]
+
+builddependencies = [
+    ('cudatoolkit/%s.85_3.18-6.0.7.0_5.1__g2eb7c52' %cudaversion, EXTERNAL_MODULE),
+    ('cray-fftw/3.3.6.5', EXTERNAL_MODULE),
+    ('cray-python/17.06.1', EXTERNAL_MODULE),
```
Comment (lucamar) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-22Aug2018-CrayGNU-18.08-cuda-9.1-pat-7.0.2.eb]: Thanks @twrobinson , please change the Python module using the following variables:
https://github.com/eth-cscs/production/wiki/UES-standards-(CUDA,-Python,-...)-for-EB-recipes#python

**Comment by twrobinson** [comment]
Retest this please

**Comment by lucamar** [comment]
The failure reported in the `logfile` is the following:
```
.depend:1015: *** multiple target patterns.  Stop.
```
However the recipe is successful when you build in your local shell.

**Comment by twrobinson** [comment]
> The failure reported in the `logfile` is the following:
> 
> ```
> .depend:1015: *** multiple target patterns.  Stop.
> ```
> 
> However the recipe is successful when you build in your local shell.

correct

**Comment by twrobinson** [comment]
test this please

**Comment by lucamar** [comment]
The error is still the same, but I assume that the recipe is working on an interactive shell:
```
.depend:1015: *** multiple target patterns.  Stop.
Makefile:175: recipe for target 'mpi' failed
make: *** [mpi] Error 2
```

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
@twrobinson Is this pull request still needed?

**Pull request closed without merging**